agent/api/eni/eni.go

```diff
@@ -171,5 +197,15 @@ func ValidateTaskENI(acsenis []*ecsacs.ElasticNetworkInterface) error {
 		return errors.Errorf("eni message validation: empty eni id in the message")
 	}
 
+	if (aws.StringValue(acsenis[0].InterfaceType) != BranchENIName) && (aws.StringValue(acsenis[0].InterfaceType) != RegularENIName) {
```

Comment:
you don't need this check i think. existing eni will not have a type and can be assumed to be standard type